### DISCUSSION TITLE
Github cli in Action not running workflow

### DISCUSSION THREAD

**Discussion by TonyWhiteSMS** [community/community#26653] [Actions]
<p>Trying to setup a scheduled workflow that triggers a workflow with particular settings on different branches (which for us equates to different projects)</p>
<pre><code class="lang-auto">- run: |
      gh workflow run 'full_branch_manual.yml' -F RunTests=true -F ValidateOnly=true  -r ${{matrix.TARGET_BRANCH}}
    env:
      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
</code></pre>
<p>But whilst the trigger runs, it fails to trigger the target workflow, it does not mark the triggering workflow with any error. I do notice that it does not output anything in the logs as a result of running the command though.</p>
<p>I have checked the doco (<a href="https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows" class="inline-onebox" rel="noopener nofollow ugc">Using GitHub CLI in workflows - GitHub Docs</a>) it says it can run any “GitHub CLI command”</p>
<p>Note I looked at Reuseable workflows but I cannot see how to specify run the workflow for a particular branch.</p>
<p>Anyone managed to achieve this? Or is this blocked to prevent loops?</p>
<p>FYI - I posted this question originally on <a href="https://stackoverflow.com/questions/71840564/github-cli-in-action-not-running-workflow" class="inline-onebox" rel="noopener nofollow ugc">Github cli in Action not running workflow - Stack Overflow</a></p>

**Comment by TonyWhiteSMS** [comment, marked as answer]
<p>Have finally tested and confirmed that creating a Personal Access Token and storing that in a Secret, and using that secret for the GITHUB_TOKEN to run the workflow does work.  A new workflow is run against the target branch, for whatever user the Personal Access Token was created against.  Just make sure it has access to run Workflows when you create the token.</p>

**Comment by otter-computer** [comment]
<p>Hey <a class="mention" href="/tonywhitesms">@TonyWhiteSMS</a>, thanks for coming back and providing an update on your question! Really appreciate you leaving the solution you found for future users that find themselves here :bowing_man:</p>
